spec/features/articles_spec.rb

```diff
@@ -0,0 +1,69 @@
+
+RSpec.feature "Articles", type: :feature do
+  before (:all) do
+    @category = FactoryBot.create(:category)
+  end
+
+  before (:each) do
+    visit root_path
+  end
+
+  let (:article) { FactoryBot.build(:article) }
+
+  scenario "unsucessfully create a new article" do
+
+    click_link "New article"
+
+    expect {
+      fill_in "Title", with: article.title
+      select 'My Awesome Category', from: 'category_id'
+      click_on "Create Article"
+    }.to change(Article.all, :count).by(0)
+
+    expect(page).to have_content "Text can't be blank"
+    expect(page).to have_content "Text is too short"
+  end
+
+  scenario "creates a new article" do
+
+    click_link "New article"
+
+    expect {
+      fill_in "Title", with: article.title
+      fill_in "Text", with: article.text
+      select 'My Awesome Category', from: 'category_id'
+      click_on "Create Article"
+    }.to change(Article.all, :count).by(1)
+
+    expect(page).to have_content article.title
+    expect(page).to have_content @category.name
+    expect(page).to have_content article.text
+  end
+
+  scenario "edit an article" do
+    article = FactoryBot.create(:article)
```

Comment:
I think you forgot this one!

Reply:
It's a create action, not a build, so it affects the DB - I don't believe it can join the let blocks above.

Reply:
Sorry, you're quite right - I hadn't noticed `create` v `build` for the others.